src/client/datascience/ipywidgets/commonMessageCoordinator.ts

```diff
@@ -136,16 +138,29 @@ export class CommonMessageCoordinator {
                     payload.moduleName,
                     payload.moduleVersion
                 );
+                this.appShell.showErrorMessage(errorMessage).then(noop, noop);
             } else if (!payload.cdnsUsed) {
+                const moreInfo = localize.Common.moreInfo();
+                const enableDownloads = localize.DataScience.enableCDNForWidgetsButton();
                 errorMessage = localize.DataScience.enableCDNForWidgetsSetting().format(
                     payload.moduleName,
                     payload.moduleVersion
                 );
+                this.appShell.showErrorMessage(errorMessage, ...[moreInfo, enableDownloads]).then((selection) => {
```

Comment:
Minor, and feel free to check with PMs. But I believe the order is off here "More Info" is always on the right of the buttons as I've seen it. Usually [commit / yes / enable, no / cancel / disable, more info].